packages/fuels-core/src/lib.rs

```diff
@@ -367,6 +367,77 @@ impl Tokenizable for fuel_tx::AssetId {
     }
 }
 
+impl<T> Tokenizable for Option<T>
+where
+    T: Parameterize + Tokenizable,
+{
+    fn from_token(token: Token) -> Result<Self, Error> {
```

Comment:
This is such a simple and brilliant way of solving the `abigen` for `Option`/`Result` 🔥